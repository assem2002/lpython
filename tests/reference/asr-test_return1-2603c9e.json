{
    "basename": "asr-test_return1-2603c9e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_return1.py",
    "infile_hash": "ec68add4e9e8059a274ba97c4349b3a8f1a43a381eb6ab330995681e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_return1-2603c9e.stderr",
    "stderr_hash": "e15b438a085131755392d90c9b2f3c760d765200844fd13ae44b6efe",
    "returncode": 2
}