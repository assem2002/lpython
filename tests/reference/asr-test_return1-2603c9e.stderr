semantic error: Type Mismatch in return, found ('f64' and 'i32')
 --> tests/errors/test_return1.py:4:5
  |
4 |     return x
  |     ^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
